public/angular/js/controllers/analysis/alg-hate-speech-graph.js

```diff
@@ -13,14 +15,14 @@ module.exports = function renderHateSpeechReportGraph(id, data, range, descripti
           d.value +
           " Reports" +
           "</div><div>" +
-          d3.timeFormat("%b. %d %H:00")(d.date).replace(/\s0/g, " ") +
+          moment(d.data.date).tz("Asia/Yangon").format('MMM. Do hh:mm:ss') +

```

Comment:
I should have said this earlier but I think since the Myanmar project is over, do we still want to make this change? We can maybe just make this change on the Myanmar server and not aggie repo. Lmk if I am missing somthing.

Reply:
LGTM